### PR TITLE
Dance Party: add potential songs for artist tweets

### PR DESCRIPTION
**Pull request by Erin007**
Follow up to #25886 

We want to avoid doing a DTP at add additional songs and artist Twitter handles if the licensing comes in late.  To do this, marketing gave me a super set of all of the potential songs to add here so that the song to artist Twitter handle mapping will be ready if needed. 

 - If the potential songs are not added, the song will never be in the dropdown so we'll never key into the hash here. 

- If the song is added at the eleventh hour, we'll be ready to Tweet at the artist without needing a DTP. 

- In the very, very slim scenario where a late song is added to the manifest, but not here, there is a default Tweet that won't include the artist handle; product team agrees this unlikely unlikely scenario is fine.

### CODE REVIEW

File: apps/src/code-studio/dancePartySongArtistTags.js [hunk 1]
```diff
@@ -33,7 +34,26 @@ export const SongTitlesToArtistTwitterHandle = {
...
+  "starships_nickiminaj" : "NickiMinaj",
+  "elprestamo_maluma" : "Maluma",
+  "despacito_luisfonsi" : "LuisFonsi",
+  "tunaktunaktun_dalermehndi" : "DalerMehndi",
+  "sapescommejamias_maitregims" : "MaitreGIMS",
+  "lesercitodelselfie_takagi" : "TakagiBeatz",
+  "gangnamstyle_psy" : "Psy_Oppa",
+  "sophia_alvarosoler" : "AsolerMusic",
+  "danzakuduro_donomar" : "DONOMAR",
+  "cestlavie_khaled" : "Khlaed_Officiel",
```
Comment (epeach) [inline apps/src/code-studio/dancePartySongArtistTags.js]: I think this might be the wrong twitter handle? Do you mean Khaled_Officiel?

Comment (Erin007) [reply apps/src/code-studio/dancePartySongArtistTags.js]: Great catch! Thanks.  I'll update here and in the gsheet.

**Comment by Erin007** [comment]
Fixed Khaled_Officiel, Sia and ArianaGrande.  Thanks for the detailed review, @epeach! Double checking that these potential songs can be out in the public; but, I don't think it should be a problem.

**Comment by Erin007** [comment]
Confirmed - these songs/Twitter handles are fine to be in the public repo.

**Pull request merged**